x/auth/client/cli/tx_sign.go

```diff
@@ -360,7 +360,7 @@ func signTx(cmd *cobra.Command, clientCtx client.Context, txF tx.Factory, newTx
 	}
 
 	if multisig != "" {
-		// Bech32 decode error, maybe it's a name, we try to fetch from keyring
+		// hex decode error, maybe it's a name, we try to fetch from keyring
```

Comment:
Ideally again we should be disabling multisig.

Reply:
The caller here is `makeMultiSignCmd` so disabling it would end up not calling this.
Let's discuss today, if we decide to disable it, we can do it. Btw, the multisig would fail in the decorator.

Reply:
Just removed multisig support [here](https://github.com/0xPolygon/cosmos-sdk/pull/3/commits/53184bac8971623a973ec75b824a7497edb2487b)